Makefile

```diff
@@ -3,6 +3,7 @@ AUTHUSERFILE ?= /var/www/vhosts/wb-thinkhazard/conf/.htpasswd
 DATA ?= world
 
 -include local.mk
+-include .env
```

Comment:
@arnaud-morvan I guess this is not the right approach. How can I pass var envs to publish target without including .env here?

Reply:
As `.env` is generated through `Makefile`, if you use it in `Makefile` it is a dependency loop.
I do not know why you want to do this, but for sure it is not the right way.

Variables in .env are passed to the containers so they are accessible :
- in publish.py or any python script with `os.environ['MYVAR']`
- in bash with `$MYVAR`.

Reply:
@arnaud-morvan Yes that makes sense. Maybe the problem lies in my target (https://github.com/GFDRR/thinkhazard/blob/8a2b3d29ceff5b1ce70170e7ec59713da213715c/Makefile#L161), but I always get the values that are exported in the Makefile as env vars in the container and not the ones in declared in .env

Reply:
Ok we just clarified that the behavior is as expected, thanks! Publish for other values than default values needs to be called with var env file currently (which will change in openshift deployment).